e2e/pages/app/page-header/page-header.testpage.component.html

```diff
@@ -1,25 +1,57 @@
 <div id="root">
 
-<ux-page-header id="pageHeader1" header="My Page" subheader="My Subheader" [crumbs]="crumbs" [items]="items"
-                [condensed]="condensed" [iconMenus]="iconMenus"></ux-page-header>
-
-<br>
-
-<button id="button1" class="btn button-primary" (click)="condensed = !condensed">Toggle Condensed</button>
-
-<ux-page-header class="m-t-xl m-b-xl" id="pageHeader2" header="My Page" subheader="My Subheader" [crumbs]="crumbs" [items]="items2"
-                [condensed]="condensed" [iconMenus]="iconMenus" [secondaryNavigation]="true"
-                [secondaryNavigationAlignment]="alignment" [secondaryNavigationAutoselect]="autoselect"></ux-page-header>
-
-<br>
-
-<button id="align-left" class="btn button-primary" (click)="alignment = 'left'">Align Left</button>
-<button id="align-center" class="btn button-primary" (click)="alignment = 'center'">Align Center</button>
-<button id="align-right" class="btn button-primary" (click)="alignment = 'right'">Align Right</button>
-
-<button id="autoselect" class="btn button-primary" (click)="autoselect = true">Enable Autoselect</button>
-
-
-<p id="selected">{{ selected }}</p>
+    <ux-page-header id="pageHeader1"
```

Comment:
There's a masthead e2e suite in the other repository, so we can just test this new feature there instead.